.ci-builds/.azure-pipelines.yml

```diff
@@ -54,10 +54,13 @@ jobs:
     - checkout: self
       lfs: true
       fetchDepth: 10
+    # Note: This job uses `make -j2` instead of `make -j -l2.5` to `make -j2` because with more parallelism
+    # this check became flaky. The build apparently was not done copying one or another .ucm file before
+    # calling makeconv for it, although the Makefile has appropriate dependencies.
```

Comment:
```suggestion
    # Note: This job uses `make -j2` instead of `make -j -l2.5` because with more parallelism (PR #2456)
    # this check became flaky. The build apparently was not done copying one or another .ucm file before
    # calling makeconv for it, although the Makefile has appropriate dependencies.
```